dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporter.java

```diff
@@ -102,30 +126,98 @@ public CompletableResultCode export(@Nonnull Collection<MetricData> metrics) {
     return export(metrics, connection);
   }
 
+  private static boolean isDeltaTemporality(AggregationTemporality temporality) {
+    return temporality == AggregationTemporality.DELTA;
+  }
+
+  private List<String> makeMetricLines(Collection<MetricData> metrics) {
+    ArrayList<String> metricLines = new ArrayList<>();
+    for (MetricData metric : metrics) {
+      boolean isDelta;
+      switch (metric.getType()) {
+        case LONG_GAUGE:
+          metricLines.addAll(serializer.createLongGaugeLines(metric));
+          break;
+        case LONG_SUM:
+          isDelta = isDeltaTemporality(metric.getLongSumData().getAggregationTemporality());
+          metricLines.addAll(serializer.createLongSumLines(metric, isDelta));
+          break;
+        case DOUBLE_GAUGE:
+          metricLines.addAll(serializer.createDoubleGaugeLines(metric));
+          break;
+        case DOUBLE_SUM:
+          isDelta = isDeltaTemporality(metric.getDoubleSumData().getAggregationTemporality());
+          metricLines.addAll(serializer.createDoubleSumLines(metric, isDelta));
+          break;
+        case SUMMARY:
+          metricLines.addAll(serializer.createDoubleSummaryLines(metric));
+          break;
+        case HISTOGRAM:
+          metricLines.addAll(serializer.createDoubleHistogramLines(metric));
+          break;
+      }
+    }
+
+    return metricLines;
+  }
+
   @VisibleForTesting
   protected CompletableResultCode export(
       Collection<MetricData> metrics, HttpURLConnection connection) {
-    String mintMetricsMessage = MetricAdapter.toMint(metrics).serialize();
-    logger.log(Level.FINEST, "Exporting: {0}", mintMetricsMessage);
+
+    List<String> metricLines = makeMetricLines(metrics);
+    String mintMetricsMessage = Joiner.on('\n').join(metricLines);
```

Comment:
we shouldn't use the term "mint" publicly as it is just the internal name for the API